static/templates/user_info_popover_content.hbs

```diff
@@ -111,11 +111,13 @@
         {{/unless}}
     {{/if}}
     {{#unless is_me}}
-    <li>
-        <a href="#" class="mention_user">
-            <i class="fa fa-at" aria-hidden="true"></i> {{#tr this}}Reply mentioning user{{/tr}} {{#if is_sender_popover}}<span class="hotkey-hint">(@)</span>{{/if}}
-        </a>
-    </li>
+        {{#if is_sender_popover}}
```

Comment:
We should avoid adding logic to the templates. Could you try to that on the `.js` corresponding file instead?

Reply:
We handle all conditions for showing/hiding different options in the popovers in templates itself. We do not do it in popover.js for any popover, so I think it should be correct to do it here to maintain consistency.

Reply:
Yeah, I think the template is the best place for this conditional.

Reply:
And we should rename this conditional to `has_message_context`; that's more correct for what it means -- if there's an associated message to reply to or not.